charts/spire/charts/spire-server/templates/configmap.yaml

```diff
@@ -32,71 +44,71 @@ server:
 
 plugins:
   DataStore:
-    - sql:
-        plugin_data:
-          {{ include "spire-server.datastore-config" . | nindent 10 }}
+    sql:
```

Comment:
As this has to do with custom plugins, I think this change is "extra".  

Also, the datastore-config is not going to be variable, I'd be happier when we get around to this to calling out the fields individually.

Reply:
Its so that it can do dictionary merging in helm. Array type values are overwritten. The macro converts it back to an array after merge.